### PR TITLE
refactor: format errors

### PR DESCRIPTION
**Pull request by e-lobo**

### CODE REVIEW

File: frappe_graphql/api.py [hunk 1]
```diff
@@ -22,6 +22,12 @@ def execute_gql_query():
         frappe.db.rollback()
         log_error(query, variables, operation_name, output)
         frappe.local.response["http_status_code"] = get_max_http_status_code(output.get("errors"))
+        errors = []
+        for err in output.errors:
+            if not isinstance(err, GraphQLError):
+                continue
```
Comment (fahimalizain) [inline frappe_graphql/api.py]: @e-lobo why skip the error completely ? 😄

**Review by fahimalizain — CHANGES_REQUESTED** [review]

File: frappe_graphql/api.py (hunk 1 above)
Comment (e-lobo) [reply frappe_graphql/api.py]: @fahimalizain fixed 👍

**Review by fahimalizain — APPROVED** [review]
👍🏼 👍🏼

File: frappe_graphql/api.py [hunk 2]
```diff
@@ -22,6 +22,12 @@ def execute_gql_query():
         frappe.db.rollback()
         log_error(query, variables, operation_name, output)
         frappe.local.response["http_status_code"] = get_max_http_status_code(output.get("errors"))
+        errors = []
+        for err in output.errors:
+            if isinstance(err, GraphQLError):
+                err = err.formatted
```
Comment (assemmarwan) [inline frappe_graphql/api.py:28]: How very nice, the structure is already prepared 😄

**Review by assemmarwan — APPROVED** [review]

**Pull request merged**